src/helpers/constants/validation.schemas.js

```diff
@@ -153,5 +153,8 @@ export default {
       .min(SLUG_MIN_LENTH)
       .max(SLUG_MAX_LENTH)
```

Comment:
rename this file to a more specific name. What are we validating here?

Reply:
I didn't work on this feature but I will reach out to the part involved to find a better way.